src/System.IO.Compression/tests/Configurations.props

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+      netstandard1.3-Unix;
+      netstandard1.3-Windows_NT;
+      netstandard1.7-Unix;
+      netstandard1.7-Windows_NT;
+      netstandard1.7;
```

Comment:
Hm, seems there's still some unnecessary configurations from the conversion I ran. I tried to clean up most of them.